src/bulma/pages/dataImport/Index.vue

```diff
@@ -67,6 +67,16 @@
             :filters="filters"
             @download-rejected="downloadRejected"
             ref="imports">
+            <template v-slot:reject="{ icon }">
```

Comment:
this can be achieved without slot, using the button configuration and an event.

we will use slot though for the new cancel line button, to make it disabled if the import is finalized.